shared/js/background/newtab-tracker-stats.js

```diff
@@ -315,9 +322,12 @@ export class NewTabTrackerStats {
             stats.push(...spliced)
```

Comment:
(Random line since GitHub doesn't allow you to leave inline comments for unchanged files.)

Nit: Maybe update the comment for this function? "Convert the internal data into..." doesn't seem accurate since it's now also external data.